core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -857,7 +857,7 @@ private void _next() throws IOException {
       val.readFields(currBlock);
 
       if (metricsGatherer != null)
-        metricsGatherer.addMetric(rk.getKey(), val);
```

Comment:
See above. The gatherer might use it.

Reply:
Fixed in c8301f2a4fa86bee62bd9c5e134add6de13091a7